docs/releases/releases.md

```diff
@@ -23,29 +25,7 @@ Using the Long Term Support version is recommended if:
 * you need a predictable migration path over a longer timeframe
 * you are not concerned about missing new features for a few years
 
-LTS releases are supported for up to 5 years:
-
-```mermaid
-%%{
-  init: {
-    'theme': 'base',
-    'themeVariables': {
-      'fontSize': '32px',
-      'primaryColor': '#8e83fe',
-      'textColor': '#555',
-      'secondaryColor': '#5952a3',
-      'tertiaryColor': 'transparent'
-    }
-  }
-}%%
-gantt
-    dateFormat  YYYY-MM-DD
-    tickInterval 12month
-    XCP-ng 8.2 LTS         :2020-11-18, 5y
-    XCP-ng 8.1 (EOL)       :2020-03-31, 1y
-```
-
-If you prefer to get the latest improvements, go for our [latest release](#-standard-release).
```

Comment:
Why not extending the Mermaid diagram with our future plans?

Reply:
Mainly because it was outdated and because the actual date for XCP-ng 8.2 EOL is a moving target right now. Other than that, no specific reason.

Reply:
I added the diagram back with approximate dates. Given the scale, this seems good enough.

Reply:
![image](https://github.com/user-attachments/assets/193db0cf-d4b3-4538-b608-594ec9dbdc9d)